android/app/src/main/java/net/minetest/minetest/Utils.java

```diff
@@ -10,7 +11,8 @@ public class Utils {
 	public static @NonNull File createDirs(File root, String dir) {
 		File f = new File(root, dir);
 		if (!f.isDirectory())
-			f.mkdirs();
+			if (!f.mkdirs())
```

Comment:
same here, if we failed to create then f has no insurance to be wrote no ?

Reply:
(see [my comment](https://github.com/minetest/minetest/pull/12960#issuecomment-1327972518))